routes.js

```diff
@@ -4,8 +4,9 @@ const fp = require('fastify-plugin')
 const path = require('path')
 
 function fastifySwagger (fastify, opts, next) {
+  const routePrefix = opts.routePrefix || '/documentation'
   fastify.route({
-    url: '/documentation/json',
+    url: `${routePrefix}/json`,
```

Comment:
if you pass `{ prefix }`  as an option, then you just have to specify `/json` as your route. See https://github.com/fastify/fastify/blob/master/docs/Plugins.md#plugin-options. Same for every other route.